rpcs3/Emu/CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ add_library(rpcs3_emu STATIC
     IPC_socket.cpp
 )
 
+set_target_properties(rpcs3_emu PROPERTIES INTERPROCEDURAL_OPTIMIZATION $<$<CONFIG:Release>:TRUE>)
```

Comment:
```suggestion
set_target_properties(rpcs3_emu PROPERTIES INTERPROCEDURAL_OPTIMIZATION $<$<OR:$<CONFIG:Release>,$<CONFIG:RelWithDebInfo>,$<CONFIG:MinSizeRel>>:TRUE>)
```

Reply:
After testing I discovered that generator expressions don't work for `INTERPROCEDURAL_OPTIMIZATION`. I just introduced an option to turn it off, because debug configurations like `Debug` and `RelWithDebInfo` do not work with LTO.

Reply:
Ah that sucks